### PR TITLE
Create a minimal template for plugin

### LINKED ISSUE #63: Add file `python/<plugin-id>.py`  to plugin folder when the template is generated.
### Description

When running the CLI command `alfasim-sdk --template` the `python/<plugin-id>.py` file was not created.

### PR DESCRIPTION
**Pull request by williamjamir**
Closes #63

### CODE REVIEW

File: alfasim_sdk/cli.py [hunk 1]
```diff
@@ -60,6 +60,11 @@ def template(dst, caption, plugin_id, author_name, author_email):
     hm = HookManGenerator(hook_spec_file_path=hook_specs_file_path)
     hm.generate_plugin_template(caption, plugin_id, author_email, author_name, dst)
 
+    source_folder = dst / plugin_id / "src"
+    python_folder = dst / plugin_id / "python"
+    python_folder.mkdir()
+    Path(source_folder / f"{plugin_id}.py").write_text("")
```
Comment (nicoddemus) [inline alfasim_sdk/cli.py]: To create an empty file you can also use `touch()` 😉

File: alfasim_sdk/cli.py [hunk 2]
```diff
@@ -70,8 +75,11 @@ def _compile(plugin_dir):
         raise FileNotFoundError(
             f"Was not possible to find a compile.py file in {plugin_dir}"
         )
+    import alfasim_sdk
 
-    subprocess.run(["python", str(compile_script)])  # nosec
+    _env = os.environ.copy()
```
Comment (nicoddemus) [inline alfasim_sdk/cli.py]: Why `_env` and not simply `env`?

File: pyproject.toml [hunk 3]
```diff
@@ -35,7 +35,8 @@ classifiers = [
 
 [tool.poetry.dependencies]
 python = "^3.6.0"
```
Comment (nicoddemus) [inline pyproject.toml]: I suggest to keep this list sorted for consistency

Comment (williamjamir) [reply pyproject.toml]: Done

File: alfasim_sdk/cli.py (hunk 1 above)
Comment (williamjamir) [reply alfasim_sdk/cli.py]: Good call =)

File: alfasim_sdk/cli.py (hunk 2 above)
Comment (williamjamir) [reply alfasim_sdk/cli.py]: Well, I just  don't have any good and valid arguments to justify this atrocity 😝

File: alfasim_sdk/cli.py [hunk 4]
```diff
@@ -58,7 +59,27 @@ def template(dst, caption, plugin_id, author_name, author_email):
     dst = Path(dst)
     hook_specs_file_path = _get_hook_specs_file_path()
     hm = HookManGenerator(hook_spec_file_path=hook_specs_file_path)
-    hm.generate_plugin_template(caption, plugin_id, author_email, author_name, dst)
+    alfasim_sdk_include = ["<alfasim_sdk_api/alfasim_sdk.h>"]
+    default_impls_for_hooks = [
+        "HOOK_INITIALIZE(ctx){" "    return 0;" "}",
```
Comment (prusse-martin) [inline alfasim_sdk/cli.py]: Why are you using 3 consecutive strings?
For all that python care those are a single continuous string.

Comment (williamjamir) [reply alfasim_sdk/cli.py]: Last commit fix this issue =)

**Pull request merged**